CHANGELOG.md

```diff
@@ -18,21 +18,21 @@ All notable changes to this project will be documented in this file.
 
 - ubi8-rust-builder: bump ubi8-minimal image to latest 8.9 ([#514]).
 - stackable-base: bump ubi8-minimal image to latest 8.9 ([#514]).
-- Bump ubi8-rust-builder toolchain to `1.74.0` ([#517]).
 - GH workflows: make preflight an independent manual workflow and update to version 1.7.2 ([#519]).
 - hadoop: Build from source ([#526]).
+- bump ubi8-rust-builder toolchain to `1.75.0` ([#542]).
```

Comment:
suggestion: I would prefer to keep both linked PRs in here. Something like:

```suggestion
- bump ubi8-rust-builder toolchain to `1.75.0` ([#542], [#517]).
```

Reply:
done in 61e1693c3d12aa4e3ef6e9b744ca3caa5cccd318